File: EzPlan_API/EzPlan/Models/Tache.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EzPlan.Models
{
    public class Tache
    {
        public int TacheID { get; set; }
        public string Nom { get; set; }
        public double DureeApproximative { get; set; }
        public double DureeMaxConsecutive { get; set; }
        public int NiveauDeStress { get; set; }
        public Utilisateur Utilisateur { get; set; }

        public Tache(){}

        public Tache(string nom, double dureeApproximative, double dureeMaxConsecutive, int niveauDeStress)
        {
            Nom = nom;
            DureeApproximative = dureeApproximative;
            DureeMaxConsecutive = dureeMaxConsecutive;
            NiveauDeStress = niveauDeStress;
        }
    }


}

File: EzPlan-Angular/src/app/models/tache.ts
export class Tache {
    tacheID?: number;
    nom?: string;
    dureeApproximative?: number;
    dureeMaxConsecutive?: number;
    niveauDeStress?: number;

    constructor () {
        this.nom = "", 
        this.dureeApproximative = 15, 
        this.dureeMaxConsecutive = 15, 
        this.niveauDeStress = 0
    }
}